src/pkgcheck/checks/pkgdir.py

```diff
@@ -121,7 +121,9 @@ def desc(self):
 
 
 class SizeViolation(results.PackageResult, results.Warning):
-    """File in $FILESDIR is too large (current limit is 20KiB)."""
+    """File in $FILESDIR is too large."""
+
+    limit = 20480
```

Comment:
let's add comment of units here for the size:
```suggestion
    limit = 20480 # bytes = 20KiB
```